slycot/__init__.py

```diff
@@ -13,7 +13,7 @@
     # import slycot.examples
 
     # Analysis routines (15/40 wrapped)
-    from .analysis import ab01nd, ab05md, ab05nd, ab07nd, ab08nd, ab08nz
+    from .analysis import ab01nd, ab04md, ab05md, ab05nd, ab07nd, ab08nd, ab08nz

```

Comment:
Please increase the count from above.

A reformat of the whle ab block seems in order, too

Reply:
#205